src/app/clusters/camera-av-stream-management-server/camera-av-stream-management-server.h

```diff
@@ -300,24 +300,17 @@ class CameraAVStreamMgmtDelegate
     virtual CHIP_ERROR OnTransportReleaseAudioVideoStreams(uint16_t audioStreamID, uint16_t videoStreamID) = 0;
 
     /**
-     * @brief Provides read-only access to the list of currently allocated video streams.
-     * This allows other components (like PushAVStreamTransportManager) to query
-     * allocated stream parameters (e.g., for bandwidth calculation) without directly
-     * accessing the CameraAVStreamMgmtServer instance.
+     * @brief Calculates the total bandwidth in Mbps for the given video and audio stream IDs.
```

Comment:
When I was talking to @aronrosenberg about the spec bits for these bandwidth measurements, he was saying that Mbps is not granular enough for real-world streams.  

In the spec all bandwidth measurements are in bps.  Why is this API doing something different?